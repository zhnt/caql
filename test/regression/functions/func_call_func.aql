function helper() {
    print("Helper function called")
}

function main_func() {
    print("Main function start")
    helper()
    print("Main function end")
}

main_func()


