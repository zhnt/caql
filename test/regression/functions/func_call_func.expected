Main function start
Helper function called
Main function end

